spec/requests/static_pages_request_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe "StaticPages", type: :request do
+
+  describe "GET /home" do
+    it "returns http success" do
+      get "/static_pages/home"
```

Comment:
```suggestion
      get :home
```
рспек автоматом підтягне необхідний роут